docs/gestor_tareas.md

```diff
@@ -0,0 +1,75 @@
+# Elección del gestor de tareas
+
+## Aspectos a tener en cuenta
+
+* Facilidad de uso
+* Comunidad
+* Mantenimiento (Freshness)
+* Velocidad
+* Funcionalidad
+
+## Opciones consideradas
+
+### Gulp
+
+Gulp es un conjunto de herramientas que ayudar a automatizar el flujo de trabajo.
+
+#### Ventajas:
+
+* Es configurable con código JavaScript.
+* Gulp permite escribir tareas individuales y componer operaciones más complejas con estas.
+* Los `plugins` son simples y están diseñados de una forma atómica.
+
+#### Inconvenientes:
+
+* Necesita `plugins`
```

Comment:
Por favor, no pongáis información totalmente irrelevante. ¿Qué más da que necesite plugins o no?